easybuild/easyconfigs/x/XMDS2/XMDS2-2.2.3-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,64 @@
+easyblock = 'Bundle'
+
+name = 'XMDS2'
+version = '2.2.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.xmds.org/'
+description = """ The purpose of XMDS2 is to simplify the process of creating simulations that
+ solve systems of initial-value first-order partial and ordinary differential equations."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('GSL', '2.5'),
+    ('libxml2', '2.9.8'),
+    ('libxslt', '1.1.32'),
+    ('lxml', '4.2.5', versionsuffix),
+    ('mpmath', '1.0.0', versionsuffix),
+    ('h5py', '2.8.0', versionsuffix),
+]
+
+exts_defaultclass = 'PythonPackage'
+exts_default_options = {
+    'download_dep_fail': True,
+    'use_pip': True,
+}
```

Comment:
This whole block can be replaced with just `use_pip = True` if the `PythonBundle` easyblock is used